src/frontend/lib/styles.js

```diff
@@ -2,7 +2,7 @@
 
 export const BLACK = "#000000";
 export const WHITE = "#ffffff";
-export const MAPEO_BLUE = "#3366ff";
+export const MAPEO_BLUE = "#0066FF";
```

Comment:
This is the updated mapeo blue according to the design specs

Reply:
was curious about what + how much this would affect but the difference in the color is pretty limited, so not as concerned

Reply:
Yeah, its very slight, that it won't be noticeable to the naked eye

<img width="1007" alt="image" src="https://user-images.githubusercontent.com/67773827/161830564-4019249d-a262-465f-9cb4-f56d1c246124.png">